src/openstudio_app/Resources/default/add_humidifier_steam_gas.rb

```diff
@@ -0,0 +1,42 @@
+# p = File.join(ENV["HOME"], "Software/Others/OSApp-build-release/OpenStudio-3.1.1/OpenStudio-3.1.1-alpha+3d83fc2b65-Linux/usr/local/openstudio-3.1.1-alpha/Ruby/openstudio.rb")
+# require p
+require 'openstudio'
+
+include OpenStudio::Model
+
+# Helper to load a model in one line
+# It will raise if the path (or the model) isn't valid
+#
+# @param path [String] The path to the osm
+# @return [OpenStudio::Model::Model] the resulting model.
+def osload(path)
+  translator = OpenStudio::OSVersion::VersionTranslator.new
+  ospath = OpenStudio::Path.new(path)
+  model = translator.loadModel(ospath)
+  if model.empty?
+      raise "Path '#{path}' is not a valid path to an OpenStudio Model"
+  else
+      model = model.get
+  end
+  return model
+end
+
+m = osload('hvac_library.osm')
```

Comment:
Is this script supposed to be run multiple times or just once?  Should it remove existing humidifiers? Seems like a useful new pattern but not quit sure what the pattern is.

Reply:
@macumber Thanks you for your contribution to OpenStudioApplication. This question is answered in our wiki at https://github.com/openstudiocoalition/OpenStudioApplication/wiki/Registering-a-new-SDK-object-in-the-OSApp#hvac-components-adding-them-to-the-hvac_library

Joke aside, this is supposed to be run (probably by me) once we have the official installer (or at least an IOFreeze version) and we bump the hvac_library to a new version. Otherwise I would be VT'ing the hvac_library now, while other VT rules haven't been added yet, and risk putting the hvac_library in a broken state. I applied the same pattern for the last release.